Source/cubfan135/script.js

```diff
@@ -31,6 +31,15 @@ if (document.cookie.length !== 0) {
 if ("serviceWorker" in navigator) {
   window.addEventListener("load", () => {
     navigator.serviceWorker.register("service-worker.js");
+    // Set Theme If Exists
+    if (localStorage.getItem("theme")) {
+      if (localStorage.getItem("theme") == "dark") {
+        darken();
+      }
+    } else {
+      // Set Theme To Default
+      localStorage.setItem("theme", "light");
+    }
```

Comment:
**This piece of code will only fire if the browser supports *Service Workers***. Some old browsers, like version 8+ of Internet Explorer, do not support *Service Workers* but they do support *Local Storage*. Keeping this code here means that users using those browsers will not be able to use this smart feature even after being able to support it.  
Move this piece of code outside of here, preferably at the end.

Reply:
Would you mind closing this PR so that I can create a new one? As you can see above, the branch I had was deleted so I had to make a new one.

Reply:
Okay, as you wish.